docs/production.md

```diff
@@ -52,7 +52,6 @@ Start the installation for CentOS 7:
 
     vagrant up centos7-katello-3.4
 
-This will create a libvirt based virtual machine running the Katello server on CentOS.
```

Comment:
Why was this removed?

Reply:
fixed in https://github.com/theforeman/forklift/pull/1185/commits/60f2326b96424c4d1faf6c644e389b4d49282844